source/zh-tw/docs/tag-plugins.md

```diff
@@ -441,3 +443,31 @@ url: https://example.com/blog # example
 content
 {% endraw %}
 ```
+
+## 文章摘要
+
+在文章中使用 `<!-- more -->`，那麼在 `<!-- more -->` 之前的文字將會被視為摘要。首頁中只出現這部份文字，同時這部份文字也會出現在文章當中。
+
+例如：
+
+```
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+<!-- more -->
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+```
+
+首頁中將只會出現
+
+```
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+```
+
+正文中則會出現
+
+```
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+
+Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+```
+
+請注意，摘要可能會被 Front Matter 中的 `excerpt` 覆寫。
```

Comment:
I found this very interesting, that we don't have this section on `zh-tw`, so I just copy the content from `zh-cn`.

Reply:
Hi @D-Sketon, would you help me check this part is correct if avaliable? Thanks!

Reply:
> so I just copy the content from `zh-cn`.

as #133, `en` is the primary language, please consider syncing content from `en` instead of other languages. Otherwise translating between different languages ​​can become confusing.

Reply:
> > so I just copy the content from `zh-cn`.
> 
> as #133, `en` is the primary language, please consider syncing content from `en` instead of other languages. Otherwise translating between different languages ​​can become confusing.

Oh, I see, no wonder I've been confused at first glance. 

I'll change this part with `en` content and also do this if I encounter to the same situation again.